src/MercadoPago.Tests/Client/Order/OrderClientTest.cs

```diff
@@ -14,6 +15,14 @@
 
     public class OrderClientTest : BaseClientTest
     {
+
+        public const string automatic = "automatic";
+        public const string manual = "manual";
+        public const string processed = "processed";
+        public const string canceled = "canceled";
+        public const string refunded = "refunded";
+        public const string partiallyRefunded = "partially_refunded";
```

Comment:
[q] as variaveis precisam ser publicas nesse caso?

```suggestion
        public const string automatic = "automatic";
        public const string manual = "manual";
        public const string processed = "processed";
        public const string canceled = "canceled";
        public const string refunded = "refunded";
        public const string partiallyRefunded = "partially_refunded";
```